test/e2e/auth/selfsubjectreviews.go

```diff
@@ -33,98 +36,131 @@ var _ = SIGDescribe("SelfSubjectReview [Feature:APISelfSubjectReview]", func() {
 	f.NamespacePodSecurityEnforceLevel = admissionapi.LevelPrivileged
 
 	/*
-		Release: v1.24
-		Testname: SelfSubjectReview API
-		Description:
-		The authentication.k8s.io API group MUST exist in the /apis discovery document.
-		The authentication.k8s.io/v1alpha1 API group/version MUST exist in the /apis/mode.k8s.io discovery document.
-		The selfsubjectreviews resource MUST exist in the /apis/authentication.k8s.io/v1alpha1 discovery document.
-		The selfsubjectreviews resource must support create.
+			Release: v1.27
+			Testname: SelfSubjectReview API
+			Description:
+			The authentication.k8s.io API group MUST exist in the /apis discovery document.
+			The authentication.k8s.io/v1alpha1 API group/version MUST exist in the /apis/mode.k8s.io discovery document.
+		    The authentication.k8s.io/v1beta1 API group/version MUST exist in the /apis/mode.k8s.io discovery document.
+			The selfsubjectreviews resource MUST exist in the /apis/authentication.k8s.io/v1alpha1 discovery document.
+			The selfsubjectreviews resource MUST exist in the /apis/authentication.k8s.io/v1beta1 discovery document.
+			The selfsubjectreviews resource MUST support create.
 	*/
 	ginkgo.It("should support SelfSubjectReview API operations", func(ctx context.Context) {
-		// Setup
-		ssarAPIVersion := "v1alpha1"
+		ginkgo.DescribeTable(
```

Comment:
It seems this can not be used inside a `ginkgo.It`block https://github.com/kubernetes/kubernetes/issues/116515#issuecomment-1466131302

Reply:
@nabokihms mind opening a fix to master to drop this call asap?

Reply:
I see. Sorry.
I will be able to fix this in 5-6 hours, not earlier.